### DISCUSSION TITLE
submit with keyboard still impossible

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [cssbattle/forum#295] [Bugs reports]
Since https://github.com/cssbattle/forum/discussions/291 has been closed prematurely: The key combination [cmd][return] still doesn’t work anymore. It submits, but adds lots of garbage line breaks and space characters before.

**Comment by Beowolve** [comment]
I can confirm this, also can we get "Ctrl+Enter" to submit instead of the Mac version with the command key, only about 10% people have mac and on windows you don't have that key. 🤷‍♂️

**Comment by chinchang** [comment]
All mentioned above issues have been fixed now. It's a shame we didn't support Ctrl+Enter for code submission until now! Please let us know if everything is good now and thanks for reporting.

**Comment by Gabb192** [comment]
What about Ctrl+Shift+Z for "REDO"? I know that also Ctrl+Y should work in the same way, but beore the editor update also Ctrl+Shift+Z was working. It would be nice to have it back for several reason :) @chinchang

**Reply by chinchang** [reply]
I have opened an issue for this @Gabb192 :) https://github.com/cssbattle/forum/issues/301
